docs/CHANGELOG.md

```diff
@@ -1,6 +1,8 @@
 ## Next
 
 * Fixed a bug where the Organizer wouldn't show all the items if your screen was very tall or zoomed out.
+* Added a sortable name label to the Compare view.
+* Sparrows no longer show their speed (all Sparrows and Skimmers have the same speed now).
```

Comment:
Bungie forgot to include the reusablePlugSetHash on them, but all sparrows can now select between 190 and 160 speed via the 2nd socket perk; would probably be worth leaving the speed in just because of that, lest someone complain that they've got them sorted by speed or something.

Reply:
OK, I hacked a speed column back in